src/clr-core/common/decorators/property.ts

```diff
@@ -35,7 +35,14 @@ export function getDefaultOptions(
     case Object:
       return { reflect: false, ...options };
     case String:
-      return { reflect: true, attribute: camelCaseToKebabCase(propertyKey), ...options };
+      return {
+        reflect: true,
+        attribute: camelCaseToKebabCase(propertyKey),
+        converter: {
+          toAttribute: (value: string) => (!!value ? value : null),
```

Comment:
If we get a falsy value remove the attribute from the element

Reply:
I may be able to reuse this now in a couple of instances in icons where i had to use setters to basically implement this logic.